homeassistant/components/sensor/moon.py

```diff
@@ -50,20 +50,20 @@ def name(self):
     def state(self):
         """Return the state of the device."""
         if self._state == 0:
-            return 'New moon'
+            return 'new moon'
```

Comment:
although we can use whitespaces in translation file keys, I'm not sure if we want this wild mix of `" ", "_". "-"` for states